spotify_project/artists/views.py

```diff
@@ -4,13 +4,13 @@
 
 
 def all_artists(request):
-    artists = Artist.objects.all().order_by(Lower('name'))
+    artists = sorted(request.user.library.artists.all(), key=lambda x: x.name.lower())
```

Comment:
This can still order by on the queryset rather than using sorted

Reply:
Technically speaking, this is now my list of artists, not the list is artists for a generic user. It's a subtle distinction that we can go over later

Reply:
I'm not understanding. You mean this is a list of the users artists not a list of generic artists? That is correct, but I'm not following why it's significant here.

Reply:
Right now, there are presumably three versions of this page. Right now, only the version of "My artists". Is written. However, it used to be "All artists". The third version would be "all artists for a given user". We can support any number of these. But the names of things will become more important the more we want to support.

For what's it's worth. We can use this view to support all three. With magic and stuff